candid/src/codegen/rust.rs

```diff
@@ -0,0 +1,225 @@
+//! Language binding for generating Rust code from a Candid file.
+//!
+//! This generates a `pub trait Actor { ... }` code for the service (if there is one) and
+//! a list of `pub type TypeX = ...` for each type declaration in a Candid file.
+//!
+//! The Actor name can be configured, as well as whether to use u128/i128 for
+//! natural/integer numbers.
+use crate::codegen::LanguageBinding;
+use crate::error::{Error, Result};
+use crate::parser::types::{Binding, Dec, FuncType, IDLType, Label, PrimType, TypeField};
+use crate::{generate_code, IDLProg};
+
+#[derive(Clone, Default)]
+pub struct Config {
+    pub actor_name: Option<String>,
+    pub use_bigint: Option<()>,
```

Comment:
While it's nice to have this option, the decoder currently only decodes to bigint. And why is the type Option<()>, not bool?